source/includes/gcp/_disks.md

```diff
@@ -0,0 +1,268 @@
+### Disks
```

Comment:
This file is here because I checked out **cmc-dev** and have written the docs for *v2* 🚀 🤦‍♂ 
So since it was in **cmc-dev** and not in **cmc-dev-v2**, its showing this.
I can remove this if need be.

Reply:
We need this doc in v1 too.

Reply:
@simongodard The base branch for this is `cmc-dev-v2`. I think it's already there in `v1` hence appearing in my branch which was off-of `v1 (cmc-dev)`. 

So if approved I can merge this with this file **(gcp_disks.md)** so that it's in v2 as well? In that case should the other gcp docs also go into v2?